source/fundamentals/indexes.txt

```diff
@@ -151,7 +151,7 @@ The following example creates an index in ascending order on the ``title`` field
    :end-before: end single index
 .. driver-content-end
 
-The following is an example of a query that would be covered by the index created above:
+The following is an example of a query that would be covered by the preceding index:
```

Comment:
```suggestion
The following is an example of a query that would be covered by the index created in the preceding code snippet:
```

Nit: I think "preceding index" changes the meaning of this sentence (makes a code snippet that creates an index equivalent to an index). I think this comment applies to the other edits on the index page as well.